client/components/nextButton.js

```diff
@@ -0,0 +1,9 @@
+import React, { PropTypes, Component } from 'react';
+
+var NextButton = (props) => (
+  <div className="nextButton"> 
+    <button onClick={props.clickHandler}>Next Page</button>
```

Comment:
Are the next/previous page buttons be <button> or <img>? Unless there is some cool styling for <button>